app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -557,9 +557,8 @@ class EditPatientScreen : BaseScreen<
 
   private fun dateOfBirthFocusChanges(): Observable<EditPatientEvent> = dateOfBirthEditText.focusChanges.map(::DateOfBirthFocusChanged)
 
-  override fun displayBpPassports(identifiers: List<BPPassportListItem>, newlyScannedIdentifiers: List<BPPassportListItem>) {
+  override fun displayBpPassports(listOfBpPassports: List<BPPassportListItem>) {
```

Comment:
Personally I would have preferred to just name it `bpPassports` 😅

Reply:
haha. oki. Changing it.